### ISSUE TITLE
Problem in IF97.h 

### ISSUE THREAD

**Issue by ibell** [CoolProp/CoolProp#1277]
Hey @henningjp, I keep getting this warning when I compile with other compilers, and it seems a little bit serious:

```
/Users/ian/Code/coolprop/externals/IF97/IF97.h:2078:9: warning: control may
      reach end of non-void function [-Wreturn-type]
        };
```
Could you please file a PR to fix this?

**Comment by henningjp** [comment]
Function had an apparent logic path that didn't return a value.  All the possible logic choices where covered, but the compiler didn't know that.  I added a default return value to the end, just in case it falls through.  Warning disappeared on VS.  Should take care of the other compilers as well.  I made a PR to the IF97 repo.

**Comment by henningjp** [comment]
This is closed with commit eaadfcf on CoolProp/IF97...
